packages/suite/src/components/wallet/InputChecksumInfo.tsx

```diff
@@ -0,0 +1,75 @@
+import styled, { css } from 'styled-components';
+import { useTheme } from 'styled-components';
+import { TooltipSymbol } from '../suite';
+import { spacingsPx, borders } from '@trezor/theme';
+import { TranslationKey, Translation } from '../suite/Translation';
+import { useTranslation } from 'src/hooks/suite';
+import { Icon } from '@suite-common/icons/src/webComponents';
+
+const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+    gap: ${spacingsPx.xs};
+    width: 100%;
+    justify-content: space-between;
+    background-color: ${({ theme }) => theme.backgroundAlertBlueSubtleOnElevation1};
+    border-radius: ${borders.radii.xxs};
+    position: relative;
+    margin: ${spacingsPx.xxs} 0 ${spacingsPx.sm} 0;
+    padding: ${spacingsPx.xs} ${spacingsPx.sm};
+    ${({ theme }) => css`
+        ::before {
+            content: '';
+            position: absolute;
+            top: -${spacingsPx.xs};
+            left: 20px;
+            transform: translateX(-50%);
+            width: 0;
+            height: 0;
+            border-left: 8px solid transparent;
+            border-right: 8px solid transparent;
+            border-bottom: 8px solid ${theme.backgroundAlertBlueSubtleOnElevation1};
+        }
+    `};
+`;
+
+const Container = styled.div`
+    display: flex;
+    flex-direction: row;
+    margin: 0;
+    padding: 0;
+`;
+
+export const Divider = styled.div`
+    height: inherit;
+    align-self: stretch;
+    width: 2px;
+    background-color: ${({ theme }) => theme.backgroundSurfaceElevation1};
```

Comment:
Just note: please use `elevation` mechanics and dont hardcode the elevation colors